### ISSUE TITLE
Staging Review finding: Some characters obscured in pseudo-alert

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#96650]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** My Education Benefits
**Product:** 22-5490 Form
**Feature:** (Fry/DEA) Updates

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/benefit-selection](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/benefit-selection), a pseudo-alert starting with "If you're the child of a veteran..." provides information about circumstances where you may be eligible for both benefits. As currently coded, some characters are partially obscured by the line beneath it --- eg. `y` looks like `v`, `,` looks like `.`, etc.

There's still enough of each character visible and clear enough context from the unobscured characters that most users will still be able to comprehend the message. But some users are likely to struggle, especially users with certain cognitive disabilities and users for whom English is a second language.

## Recommended action

I think this is solved easily by using a `va-alert` component.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
